Source/items.cpp

```diff
@@ -3794,11 +3794,11 @@ void PrintItemPower(char plidx, ItemStruct *x)
 		break;
 	case IPL_LIFE:
 	case IPL_LIFE_CURSE:
-		sprintf(tempstr, "Hit Points: %+i", x->_iPLHP >> 6);

```

Comment:
`_iPLHP` and `_iPLMana` use the low 6 bits as the "fractional" part.
We should add `int_hp()` and `int_mana()` methods rather than changing the bit shift to division.